src/LearnositySdk/Request/Init.php

```diff
@@ -140,6 +141,39 @@ public function __construct($service, $securityPacket, $secret, $requestPacket =
         $this->securityPacket['signature'] = $this->generateSignature();
     }
 
+    /**
+     * Adds metadata to request packet to enable telemetry and tracking. Request packet will be
+     * extended with following parameters:
+     * {
+     *   "meta": {
+     *     "sdk": {
+     *       "version": "v1.2.1",
+     *       "lang": "java",
+     *       "lang_version": "oracle-v1.9.0",
+     *       "platform": "linux",
+     *       "platform_version": "10"
+     *     }
+     *   }
+     * }
+     *
+     * @param mixed    $requestPacket
+     *
+     * @return mixed
+     */
+    private function addMeta($requestPacket) {
+        $requestPacket['meta'] = [
+            'sdk' => [
+                'version' => 'hardcoded for now',
```

Comment:
This I will work on tomorrow ;)

Reply:
For this, my suggestions would be:
1. Make the name of the file a constant somewhere
2. Make a specific method to get the SDK version that would
2.1 Test that the file exists, or return a sane value
2.2 Simply returns a `file_get_contents` of that file, assuming it contains only the version number, in the desired format